weekly/2023/week-45.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week 45 in Packit
+date: 2023-11-13
+authors: mfocko
+tags:
+  - 2023-November
+  - 2023
+  - November
+---
+
+## Week 45 (November 7th – November 13th)
+
+- Now Packit allows `~^+` (special characters) in a tag (used for building the version string). ([packit#2154](https://github.com/packit/packit/pull/2154))
+- The downstream Koji builds are now stored in the database as well and exposed via `/api/koji-builds` endpoint. ([packit-service#2244](https://github.com/packit/packit-service/pull/2244))
+- Downstream Koji builds are now shown in a separate table in `/jobs/downstream-koji-builds`. ([packit-service#2249](https://github.com/packit/packit-service/pull/2249), [dashboard#354](https://github.com/packit/dashboard/pull/354))
```

Comment:
these should be probably merged together, we can mention only the dashboard and API